paper.md

```diff
@@ -151,40 +151,43 @@ bibliography: paper.bib
-control (and metadata). ATM this follows implicitly but may be much less obvious
-to someone who didn't dive into it yet -->
-
-## Why Git and git-annex alone are not enough
-
-<!-- this is "what does datalad add to Git+git-annex -->
+Code, data, and computing environments are at the core of scientific practice.
+Unobstructed access and efficient management of all those digital objects promotes scientific discovery through collaboration, reproducibility, and replicability.
+Established and widely-adopted infrastructure and procedures, like software distributions, distributed version control systems, and social coding portals like GitHub, streamline the collaborative development and use of research software.
+However, data have remained a “2nd-class citizen” in the contemporary scientific process, despite the increasing embracement of the FAIR principles [@FAIR2016].
+1) Disconnected data portals force a cacophony of data access and authentication methods upon the scientific community; 2) data versioning is rarely performed, or nowhere close to the precision and inspectability that is standard in software development; and 3) data provenance---if shared at all---is often not complete, because their capture is rarely considered to be an integral part of the scientific process.
+DataLad aims to solve all these issues by providing targeted interfaces and interoperability adapters to standard tools and services, scientific and commercial.
+It enables workflows that are particularly suited for reproducible science, such as actionable process provenance capture for arbitrary command execution that affords automatic re-execution.
+In order to maximize its utility and target audience, DataLad is available for all major operating systems, and can be integrated into established workflows and environments with minimal adjustments.
+
+## Why Git and git-annex?
+
+Git is the standard version control system for software development[^1].
+It is a distributed content management system, specifically tuned towards managing and collaborating on text files, and excels at making all content committed to Git reliably and efficiently available to all clones of a repository.
+At the same time, Git is not designed to efficiently handle large (e.g., over a gigabyte) or binary files [see, e.g., @opensource:git-binary].
+This makes it hard or impossible to use Git directly for distributed data storage with tailored access to individual files, or even to fully remove file content without severely impacting the integrity of a repository.
+Git-annex takes advantage of Git's ability to efficiently manage textual information to overcome this limitation.
+File content managed by git-annex is placed into a managed repository annex, instead of committing it directly
+to Git.
+Instead of the file content, git-annex only commits a compact reference that enables identification and association of file name and content.
```

Comment:
```suggestion
Instead of the file content, git-annex only commits a compact reference that enables identification and association of a file name with the content.
```
since it is not necessarily a reversible relationship, and content might be associated with multiple files